sbncode/EventGenerator/MeVPrtl/Tools/PDGCodes.h

```diff
@@ -0,0 +1,296 @@
+//____________________________________________________________________________
+/*!
+
+  \file     PDGCodes.h
+
+  \brief    Most commonly used PDG codes.
+  A set of utility functions to handle PDG codes is provided in PDGUtils
+
+  \author   Costas Andreopoulos <constantinos.andreopoulos \at cern.ch>
+  University of Liverpool & STFC Rutherford Appleton Laboratory
+
+  Changes required to implement the GENIE Boosted Dark Matter module
+  were installed by Josh Berger (Univ. of Wisconsin)
+
+  \created  May 06, 2004
+
+  \cpright  Copyright (c) 2003-2020, The GENIE Collaboration
+  For the full text of the license visit http://copyright.genie-mc.org           
+
+*/
+//____________________________________________________________________________
+
+#ifndef _PDG_CODES_H_
+#define _PDG_CODES_H_
+
+namespace genie {
```

Comment:
Thanks for adding this in! This is a good idea. Although I would suggest changing the namespace from genie to mevprtl (or something like that) and to edit the comment at the top to reflect that this file is not being used in GENIE here.